Index: renderer/src/components/start/App.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"use client\";\r\n\r\nimport { Flex } from \"@chakra-ui/react\";\r\nimport { SelectProgramForStartLogic } from \"@/components/start/SelectProgramForStartLogic\";\r\nimport { StartButtonLogic } from \"@/components/start/StartButtonLogic\";\r\nimport { OptionCommon } from \"@/components/start/OptionCommon\";\r\nimport { commonStorage, nShoppingStorage } from \"@/util/localStorage\";\r\nimport { useEffect } from \"react\";\r\n\r\nexport const App = () => {\r\n  useEffect(() => {\r\n    // localStorage의 값을 valtio store에 동기화\r\n    nShoppingStorage.syncToValtio();\r\n    commonStorage.syncToValtio();\r\n  }, []);\r\n\r\n  return (\r\n    <Flex direction={\"column\"} gap={9} p={5}>\r\n      <OptionCommon />\r\n      <SelectProgramForStartLogic />\r\n      <StartButtonLogic />\r\n    </Flex>\r\n  );\r\n};\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/renderer/src/components/start/App.tsx b/renderer/src/components/start/App.tsx
--- a/renderer/src/components/start/App.tsx	
+++ b/renderer/src/components/start/App.tsx	
@@ -4,16 +4,9 @@
 import { SelectProgramForStartLogic } from "@/components/start/SelectProgramForStartLogic";
 import { StartButtonLogic } from "@/components/start/StartButtonLogic";
 import { OptionCommon } from "@/components/start/OptionCommon";
-import { commonStorage, nShoppingStorage } from "@/util/localStorage";
 import { useEffect } from "react";
 
 export const App = () => {
-  useEffect(() => {
-    // localStorage의 값을 valtio store에 동기화
-    nShoppingStorage.syncToValtio();
-    commonStorage.syncToValtio();
-  }, []);
-
   return (
     <Flex direction={"column"} gap={9} p={5}>
       <OptionCommon />
